src/dashboards/vpc-flow-logs.json

```diff
@@ -412,6 +417,31 @@
         "queryValue": "",
         "skipUrlSync": false,
         "type": "custom"
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "vpc_flow_logs",
+          "value": "vpc_flow_logs"
+        },
+        "datasource": {
+          "type": "grafana-athena-datasource",
+          "uid": "${datasource}"
+        },
+        "definition": "",
+        "hide": 0,
+        "includeAll": false,
+        "multi": false,
+        "name": "Database",
```

Comment:
can you move this variable to before table variable? (so we have data source > database > table )

Also, for the tables variable, can you change it to `show tables from $Database` so it uses the one from the selector.

You can also do this for the CUR dashboard.